src/lib/chips/chip.ts

```diff
@@ -147,13 +147,13 @@ export class MatChip extends _MatChipMixinBase implements FocusableOption, OnDes
   chipListSelectable: boolean = true;
 
   /** The chip avatar */
-  @ContentChild(MatChipAvatar) avatar: MatChipAvatar;
+  @ContentChild(MatChipAvatar) avatar!: MatChipAvatar;
```

Comment:
Optional since there isn't always an avatar?
(same for other ChildrenChild instances here)